.github/workflows/TagBot.yml

```diff
@@ -4,6 +4,23 @@ on:
     types:
       - created
   workflow_dispatch:
+    inputs:
+      lookback:
+        default: "3"
```

Comment:
```suggestion
```
Lookback is already 3 by default, no? And IIUC this input will be unused because it isn't passed to the TagBot action.

Reply:
No, these three lines are about adding to the GutHub UI for manual workflow dispatch an input field that allows overriding the lookback value. That's helpful if TagBot ever again fails and you don't notice within 3 days

Reply:
Yes it has. This is how these workflow dispatch inputs work (and I used it like that several times yesterday and today, too)

Reply:
Would you be willing to share a link to the docs that describe this implicit argument passing behavior so I can understand it, too?

Reply:
I would imagine this would be necessary: https://github.com/JuliaRegistries/TagBot/pull/389